pkg/controller/realmadmin/settings.go

```diff
@@ -105,14 +106,6 @@ func (c *Controller) HandleSettings() http.Handler {
 			return
 		}
 
-		// SMS config is all or nothing
-		if (form.TwilioAccountSid != "" || form.TwilioAuthToken != "" || form.TwilioFromNumber != "") &&
```

Comment:
Moved this into the model in preparation for having a system admin page to create Twilio credentials.